packages/expo-constants/android/src/main/java/expo/modules/constants/ConstantsService.java

```diff
@@ -111,20 +111,11 @@ public ConstantsService(Context context) {
     return constants;
   }
 
-  @Deprecated
-  public String getAppId() {
```

Comment:
This was my error in https://github.com/expo/expo/pull/12964. I forgot to change the base versioned code but changed the versions. This PR updates the base versioned code and 42 versioned code to match.